src/InsertionsClient/ConsoleApp/Program.cs

```diff
@@ -147,9 +183,11 @@ private static void ShowHelp()
             Console.WriteLine($"{SwitchDefaultConfig}   full path on disk to default.config to update");
             Console.WriteLine($"{SwitchManifest}   full path on disk to manifest.json");
             Console.WriteLine($"{SwitchIgnorePackages}   full path on disk to ignored packages file. Each line should have a package id [optional]");
+            Console.WriteLine($"{SwitchPropsFilesRootDir}   directory to search for and update .props files [optional]");

```

Comment:
minor: please note there are two blank spaces in these statements.

Reply:
I am seeing 3 spaces in all the other lines. Should I replace them with 2?